spec/firebase_test_lab_result_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+EMPTY_FIREBASE_TEST_LOG_PATH = File.join(__dir__, 'test-data', 'empty.json')
+PASSED_FIREBASE_TEST_LOG_PATH = File.join(__dir__, 'test-data', 'firebase', 'firebase-test-lab-run-passed.log')
+FAILED_FIREBASE_TEST_LOG_PATH = File.join(__dir__, 'test-data', 'firebase', 'firebase-test-lab-run-failure.log')
+INVALID_TEST_LOG_PATH = 'foo'.freeze
```

Comment:
Same, you should convert those to `let(:…) { … }`

Reply:
This is a bit weird, because I want to share them between all tests – I adjusted it in https://github.com/wordpress-mobile/release-toolkit/commit/5d15b3a8a8defa7b83068034e59a3491b92f14c9, interested to hear your thoughts?